packages/gatsby-recipes/src/graphql-server/index.js

```diff
@@ -0,0 +1,56 @@
+import execa from "execa"
+import * as path from "path"
+import * as fs from "fs"
+import detectPort from "detect-port"
+import {
+  getService,
+  createServiceLock,
+} from "gatsby-core-utils/dist/service-lock"
+
+// NOTE(@mxstbr): The forceStart boolean enforces us to start the recipes graphql server
```

Comment:
Could this result in having two separate recipe servers running?

Reply:
Yes, but that's fine temporarily I think. cc @johno for confirmation

Reply:
Confirmed by johno in Slack!